en/docs/api-gateway/next/gateway-controller-management-api/llm-provider-template-management.md

```
@@ -75,6 +75,20 @@ Add a new LLM provider template to the Gateway. A template defines token trackin
 }
 ```
 
+!!! note "Using a YAML downloaded from AI Workspace"
+    If you downloaded the template YAML from AI Workspace, you have two options to use it with this endpoint:
+
+    - **Convert to JSON:** Convert the YAML file to JSON before passing it as the request body with `-H 'Content-Type: application/json'`.
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)